src/lib/binutils/RelocateCubin.cpp

```diff
@@ -247,6 +247,26 @@ applyLineMapRelocations
 }
 
 
+static void
+applyDebugInfoRelocations
```

Comment:
Since we are no longer using these just for DebugInfo, the name should change to 

applyRELArelocations